.github/workflows/build_package.yml

```diff
@@ -67,35 +71,56 @@ jobs:
             build-family: macos
             build-package: py-runtime-pkg
             experimental: true
+
+          # Windows packages.
+          - runs-on:
+              - ${{ github.repository == 'openxla/iree' && 'windows-2022-64core' || 'windows-2022'}}
+            build-family: windows
+            build-package: py-compiler-pkg
+            experimental: true
+          - runs-on:
+              - ${{ github.repository == 'openxla/iree' && 'windows-2022-64core' || 'windows-2022'}}
```

Comment:
If you want to save some money, I don't think the 64core machine will buy much for the runtime build. It is mostly clone/configure time, ime.

Reply:
Good point... the smaller machines are actually faster too :sweat_smile: 

Smaller Windows machine:
https://github.com/ScottTodd/iree/actions/runs/5765014766/job/15630135942
7m 54s (2m40s checkout, 0m10s deps, 4m40s build)

Larger Windows machine:
https://github.com/openxla/iree/actions/runs/5765425296/job/15631341504
11m 55s (8m16s checkout, 1m30s deps, 1m40s build)

I can make that switch tonight/tomorrow.